package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "devDependencies": {},
   "dependencies": {
-    "xdl": "^44.0.2"
+    "xdl": "^44.0.2",
+    "expo-detox-hook": "^1"
```

Comment:
where does this expo-detox-hook package live?

Reply:
here: https://github.com/expo/detox-tools